apps/navstar_overlay/src/navstar_overlay_configure.erl

```diff
@@ -0,0 +1,105 @@
+%%%-------------------------------------------------------------------
+%%% @author sdhillon
+%%% @copyright (C) 2016, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 27. May 2016 9:27 PM
+%%%-------------------------------------------------------------------
+-module(navstar_overlay_configure).
+-author("sdhillon").
+-author("dgoel").
+
+%% API
+-export([start/2, stop/1, maybe_configure/1, maybe_configure_and_reply/2]).
+
+-include_lib("mesos_state/include/mesos_state_overlay_pb.hrl").
+
+start(reply, LashupEvent) ->
```

Comment:
These should probably be named start_link for consistency. Also, why not always reply?